src/common/internal/test_group.ts

```diff
@@ -536,7 +536,12 @@ class RunCaseSpecific implements RunCase {
                       // Prepend the subcase name to all error messages.
                       for (const arg of args) {
                         if (arg instanceof Error) {
-                          arg.message = subcasePrefix + '\n' + arg.message;
+                          try {
+                            arg.message = subcasePrefix + '\n' + arg.message;
+                          } catch {
+                            // Silence exception if the property isn't settable
+                            // (e.g. arg.message on DOMException).
+                          }
```

Comment:
Honestly I'm not sure what this try-catch is for. Does setting non-settable (read-only) property throw an exception? But it doesn't although I tried the following example code on my Chrome console.

```
const e = new DOMException('aaa');
try {
  e.message = 'bbb' + e.message;
  console.log('no exception');
} catch {
  console.log('exception');
}
console.log(e.message);

Output:
no exception
aaa // read-only so no change
```

Reply:
Ah, ok TypeScript(?) seems to check it so the change makes sense to me.

![image](https://user-images.githubusercontent.com/7637832/210485069-d79cf9ef-ed49-4409-8813-f7dacc3c3e72.png)

Reply:
There is an exception when `"use strict";` is enabled. I got confused by this too :)